pkg/image/trigger/buildconfigs/buildconfigs.go

```diff
@@ -144,7 +144,7 @@ func (r *buildConfigReactor) ImageChanged(obj runtime.Object, tagRetriever trigg
 			continue
 		}
 		if p.Paused {
-			glog.V(5).Infof("Skipping paused build on bc: %s/%s for trigger: %s", bc.Namespace, bc.Name, t)
+			glog.V(5).Infof("Skipping paused build on bc: %s/%s for trigger: %+v", bc.Namespace, bc.Name, t)
```

Comment:
the trigger is full of pointers. Does it actually look helpful?

Reply:
it is up to @bparees but I guess it will at least print the type ;-)

Reply:
probably fine.